include/mbedtls/ssl_internal.h

```diff
@@ -256,6 +256,22 @@
       + ( MBEDTLS_SSL_CID_OUT_LEN_MAX ) )
 #endif
 
+#if defined(MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH)
```

Comment:
In commit "Add static functions to calculate ssl buffer sizes ", we say "Add two, so far unused functions to calculate the input and output ssl buffer sizes." Instead, explain what they'll be used for if they deserve to be added in their own commit, or move their addition to the commit that uses them (that commit would also contain their justification already).